QUEST_LV_0200.tsv

```diff
@@ -1127,13 +1127,13 @@ QUEST_LV_0200_20150317_001126	There are many like that. {nl}I think people in ou
 QUEST_LV_0200_20150317_001127	Please help me. {nl}I started my business with all that I've got and now I'm about to end up with nothing.
 QUEST_LV_0200_20150317_001128	Oh how can I thank you.. These sure are my stuff. {nl}I did think it was weird that you were fine unlike other Revelators. You truly are the special one.
 QUEST_LV_0200_20150317_001129	Villager Emil
-QUEST_LV_0200_20150317_001130	I need to go downtown but the priest says the seal is dangerous and I should stay put. {nl}That seal really is nothing for me but the demons are scary.
-QUEST_LV_0200_20150317_001131	I guess there's nothing I can do in order to live. {nl}Although all the villagers here are just looking up to the Goddess and priest.
-QUEST_LV_0200_20150317_001132	I saw other Revelators all gone weird but you seem fine. Did you meet the Goddess or something? {nl}Anyways, that's awesome. I want to be as strong as you are.
+QUEST_LV_0200_20150317_001130	$I used to travel in and out of town, but the priest told me to stay put, saying the seal is dangerous.{nl}That seal really is nothing for me, it's the demons that are scary.
+QUEST_LV_0200_20150317_001131	$I have to have a livelihood. {nl}The other villagers here just spend all day looking up to the priest and Goddess.
+QUEST_LV_0200_20150317_001132	$I've seen other Revelators here that got all weird but you seem fine.{nl}Did you meet the Goddess or something? {nl}Anyways, you did some awesome work. I wish I was as strong as you are.
```

Comment:
the story says they are weird beyond just behavior. for example they poof with weird noises when approached